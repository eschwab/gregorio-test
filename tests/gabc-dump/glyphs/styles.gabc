%notes: styles, verb and special characters
initial-style: 0;
%%
{a}<i>i</i><b>b</b><ul>ul</ul><sp>+</sp><v>a</v>(gZ)
{a}<a<sp>+</sp><sp>%</sp>><v><</v><v>></sp></v>(fZ)
