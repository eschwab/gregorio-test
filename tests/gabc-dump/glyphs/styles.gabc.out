=====================================================================
 SCORE INFOS
=====================================================================
   number_of_voices          1


=====================================================================
 VOICES INFOS
=====================================================================
  Voice 1
   initial_key               5 (c3)


=====================================================================
 SCORE
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style    ST_FORCED_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style          ST_FORCED_CENTER
---------------------------------------------------------------------
     beginning of style          ST_ITALIC
---------------------------------------------------------------------
     character                 i
---------------------------------------------------------------------
     end of style                ST_ITALIC
---------------------------------------------------------------------
     beginning of style            ST_BOLD
---------------------------------------------------------------------
     character                 b
---------------------------------------------------------------------
     end of style                  ST_BOLD
---------------------------------------------------------------------
     beginning of style      ST_UNDERLINED
---------------------------------------------------------------------
     character                 u
---------------------------------------------------------------------
     character                 l
---------------------------------------------------------------------
     end of style            ST_UNDERLINED
---------------------------------------------------------------------
     beginning of style    ST_SPECIAL_CHAR
---------------------------------------------------------------------
     character                 +
---------------------------------------------------------------------
     end of style          ST_SPECIAL_CHAR
---------------------------------------------------------------------
     beginning of style        ST_VERBATIM
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style              ST_VERBATIM
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  g
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    8 (GRE_END_OF_LINE)
     element_type            13 (GRE_END_OF_PAR)
=====================================================================
   type                      11 (GRE_SYLLABLE)
   position                  4 (WORD_ONE_SYLLABLE)
---------------------------------------------------------------------
     beginning of style    ST_FORCED_CENTER
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     end of style          ST_FORCED_CENTER
---------------------------------------------------------------------
     character                 <
---------------------------------------------------------------------
     character                 a
---------------------------------------------------------------------
     beginning of style    ST_SPECIAL_CHAR
---------------------------------------------------------------------
     character                 +
---------------------------------------------------------------------
     end of style          ST_SPECIAL_CHAR
---------------------------------------------------------------------
     beginning of style    ST_SPECIAL_CHAR
---------------------------------------------------------------------
     character                 %
---------------------------------------------------------------------
     end of style          ST_SPECIAL_CHAR
---------------------------------------------------------------------
     character                 >
---------------------------------------------------------------------
     beginning of style        ST_VERBATIM
---------------------------------------------------------------------
     character                 <
---------------------------------------------------------------------
     end of style              ST_VERBATIM
---------------------------------------------------------------------
     beginning of style        ST_VERBATIM
---------------------------------------------------------------------
     character                 >
---------------------------------------------------------------------
     character                 <
---------------------------------------------------------------------
     character                 /
---------------------------------------------------------------------
     character                 s
---------------------------------------------------------------------
     character                 p
---------------------------------------------------------------------
     character                 >
---------------------------------------------------------------------
     end of style              ST_VERBATIM
---------------------------------------------------------------------
     type                    3 (GRE_ELEMENT)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       type                  2 (GRE_GLYPH)
       glyph_type            16 (G_PUNCTUM)
-  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  
         type                   1 (GRE_NOTE)
         pitch                  f
         shape                  1 (S_PUNCTUM)
---------------------------------------------------------------------
     type                    8 (GRE_END_OF_LINE)
     element_type            13 (GRE_END_OF_PAR)
=====================================================================
